src/components/Card.js

```diff
@@ -0,0 +1,20 @@
+const Card = ({ first_name, last_name, avatar, employment }) => {
```

Comment:
It is better to use camelcase for first_name and last_name as it is the convention in javascript.

Reply:
The properties from the API are named like this, so I decide to use them like that! do you suggest renaming those properties while destructuring?

Reply:
You can rename them as you are destructuring them